recipes/timsort/all/test_package/CMakeLists.txt

```
@@ -5,4 +5,3 @@ find_package(gfx-timsort REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
 target_link_libraries(${PROJECT_NAME} PRIVATE gfx::timsort)
```

```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE gfx::timsort)
target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_20)
```

The C++ standard is not passed when building with Conan 1.x, because compiler.cppstd is None by default. In Conan 2.x, Conan will pass the standard to CMake via CMAKE_CXX_STANDARD definition.

---

No problem for the delay, I know that you're quite busy handling a bunch of different recipes 🙂 
Is it sufficient to ensure that all versions work correctly? Only the 3.0.0 requires C++20, older versions are supposed to be compatible with older standards. Or is it just best effort?

---

Yes. Like I commented before, Conan 2.x will the C++ standard based on compiler.cppstd.